plejd/ble.bluez.js

```diff
@@ -85,8 +85,9 @@ class PlejdService extends EventEmitter {
       this.objectManager.removeAllListeners();
     }
 
+    this.bleDevices = [];
```

Comment:
We need to reset this to not continue the list on a re-init (write/ping errorhandler)